README.md

```diff
@@ -1,103 +1,79 @@
-
-### lts-and-current images
-
-Gradle's support for new Java releases historically has lagged for multiple months.
-This means most users wanting to use the latest Java release will need to do so using toolchains.
-Toolchains are documented [here](https://docs.gradle.org/current/userguide/toolchains.html) and [here](https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#configuration-toolchains) for GraalVM.
-The lts-and-current images provide both the latest LTS JDK and the latest (LTS or non-LTS) JDK.
-This allows Gradle to be launched with a supported JDK (the latest LTS release)
-and configure the compilation using toolchains to use the latest current JDK.
-This is done by putting the content below in `/home/gradle/.gradle/gradle.properties`.
+# Gradle Docker Image
 
-```properties
-org.gradle.java.installations.auto-detect=false
-org.gradle.java.installations.auto-download=false
-org.gradle.java.installations.fromEnv=JAVA_LTS_HOME,JAVA_CURRENT_HOME
-```
+[![Project Status: Active](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](/LICENSE)
+[![Build status](https://github.com/gradle/docker-gradle/workflows/GitHub%20CI/badge.svg)](https://github.com/gradle/docker-gradle/actions?query=workflow%3A%22GitHub+CI%22)
 
-The `JAVA_LTS_HOME` environment variable points to the path
-where the latest LTS JDK is installed and `JAVA_CURRENT_HOME` points to the latest current JDK.
-These may point to the same path if the latest JDK is an LTS release.
+The official Docker image for [Gradle](https://gradle.org/).
+Maintained by the [Gradle team](https://github.com/gradle/docker-gradle) as an [Official Image](https://github.com/docker-library/official-images). Thanks to [@keeganwitt](https://github.com/keeganwitt) for his years of stewardship.
 
-## What is Gradle?
+[Gradle](https://gradle.org/) is a fast, dependable, and adaptable open-source build automation tool with an elegant and extensible declarative build language.
 
-[Gradle](https://gradle.org/) is a build tool with a focus on build automation and support for multi-language development. If you are building, testing, publishing, and deploying software on any platform, Gradle offers a flexible model that can support the entire development lifecycle from compiling and packaging code to publishing websites. Gradle has been designed to support build automation across multiple languages and platforms including Java, Scala, Android, C/C++, and Groovy, and is closely integrated with development tools and continuous integration servers including Eclipse, IntelliJ, and Jenkins.
+## Supported Tags
 
-## How to use this image
+- **JDK 8** → [`jdk8`, `jdk8-jammy`](jdk8-jammy/Dockerfile), [`jdk8-corretto`](jdk8-corretto/Dockerfile), [`jdk8-ubi9`](jdk8-ubi9/Dockerfile)
```

Comment:
Java 8 images do not exist on `master` any more, we should probably make them absolute and point to the `8` branch like `https://github.com/gradle/docker-gradle/blob/8/jdk8-jammy/Dockerfile`

Reply:
Fixed